### PR TITLE
Nativo: Added User Sync settings

### PR DESCRIPTION
**Pull request by rafataveira**

### CODE REVIEW

**Review by AntoxaAntoxic — APPROVED** [review]

File: src/main/resources/bidder-config/generic.yaml [hunk 1]
```diff
@@ -62,6 +62,13 @@ adapters:
            - native
           supported-vendors:
           vendor-id: 263
+        usersync:
+          enabled: true
```
Comment (SerhiiNahornyi) [inline src/main/resources/bidder-config/generic.yaml:66]: by default, enabled property has `true` value, so pls remove `enabled: true`

Comment (rafataveira) [reply src/main/resources/bidder-config/generic.yaml]: done

**Review by AntoxaAntoxic — APPROVED** [review]

**Review by And1sS — CHANGES_REQUESTED** [review]
Pls, add `enabled: true` or `enabled: false` to added `usersync` section. Thats why pipeline fails.

**Comment by SerhiiNahornyi** [comment]
> Pls, add `enabled: true` or `enabled: false` to added `usersync` section. Thats why pipeline fails.

It is our internal bug that we don't support this default parameter, no changes needed for this PR

**Comment by rafataveira** [comment]
@SerhiiNahornyi Thank you for reviewing this. Is there any timeline on when this PR will be merged?

**Comment by SerhiiNahornyi** [comment]
> @SerhiiNahornyi Thank you for reviewing this. Is there any timeline on when this PR will be merged?

In a week, maybe earlier, we will have the bug fixed. 
As soon as it is fixed we will merge your PR.

**Comment by SerhiiNahornyi** [comment]
Fix for mentioned bug
https://github.com/prebid/prebid-server-java/pull/2936

**Pull request merged**